unic/char/range/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "unic-char-range"
+version = "0.5.0"
+authors = ["The UNIC Project Developers"]
+repository = "https://github.com/behnam/rust-unic/"
+license = "MIT/Apache-2.0"
+keywords = ["text", "unicode", "iteration"]
+description = "UNIC - CharRange"
+categories = ["text-processing", "iteration"]
```

Comment:
There's no "iteration" category. Maybe we don't need more than one. Here's the list: https://crates.io/categories/

FYI, I've got "internationalization" (and "localization") added to the list, but it hasn't made it to the live version yet. When that happens, almost all UNIC components get both `text-processing` and `internationalization`.